cli_sdk.py

```diff
@@ -84,14 +84,17 @@ def get_response(intent, transcript, raw_data=False):
     return return_json_data(payload, raw_data)
 
 
-def get_entities(intent, transcript):
+def get_entities(intent, transcript, raw_data=False):
```

Comment:
I think it may be more intuitive to have two functions instead of changing the return value based on an argument. Something like `get_entities` and `print_entities`.

Reply:
But if this flag is only for development purposes then we probably don't need to make two functions.

Reply:
The design paradigm in place throughout cli_sdk.py is to have a kwarg that allows for optional returns. I'd be inclined to either keep the current design pattern or apply your suggestion to all of them.